presto-main/src/main/java/com/facebook/presto/FullConnectorSession.java

```diff
@@ -53,13 +53,13 @@ public FullConnectorSession(Session session)
     public FullConnectorSession(
             Session session,
             Map<String, String> properties,
-            ConnectorId connectorId,
+            CatalogName catalogName,
             String catalog,
             SessionPropertyManager sessionPropertyManager)
     {
         this.session = requireNonNull(session, "session is null");
         this.properties = ImmutableMap.copyOf(requireNonNull(properties, "properties is null"));
-        this.connectorId = requireNonNull(connectorId, "connectorId is null");
+        this.catalogName = requireNonNull(catalogName, "connectorId is null");
```

Comment:
update message... maybe search codebase for "connectorId" since we shouldn't really have that anymore.